### ISSUE TITLE
The Fix "Fix support for python 3" (commit 97ead19db2218ea0b3c2f19aca8f519d97c19b9f) broke Django Tutorial

### ISSUE THREAD

**Issue by carlsoane** [mathebox/django_hana_pyhdb#22]
#### What is the expected behavior?
I was running the [Django project tutorial](https://docs.djangoproject.com/en/1.10/intro/tutorial01/),  against HANA Express (HXE) because we are trying to highlight Python usage with HXE for the next release of HXE. 

The problem occurs when running the python manage.py migrate command in the "Activating Models" [section ](https://docs.djangoproject.com/en/1.10/intro/tutorial02/). The problematic command is: `python manage.py migrate`.  I get an error concluding with:   

_File "C:\dev\python\djangopure\lib\site-packages\pyhdb\cursor.py", line 101, in __next__
    (len(self._params_metadata), len(parameters)))
django.db.utils.DatabaseError: Prepared statement parameters expected 3 supplied 6._

I am able to complete the tutorial when I build my django-hana-1.1.py2.7.egg using this commit:
commit a94a5ac0128a89f9ecdfbcc6f0ca2f5d2a9228bc
Author: Max Bothe <mathebox@gmail.com>
Date:   Sat Feb 4 21:11:04 2017 +0100

    Add testing framework and travis support (#4)

But the tutorial fails when I build using the next commit:
commit 97ead19db2218ea0b3c2f19aca8f519d97c19b9f
Author: Max Bothe <mathebox@gmail.com>
Date:   Sat Feb 4 23:38:05 2017 +0100

    Fix support for python 3 (#5)

I traced the problem to the following changes in DatabaseOperations class in operations.py:

    def modify_update_params(self, params):
        #CSOANE: Had to reverse the change from map to tuple in
        # commit 97ead19db2218ea0b3c2f19aca8f519d97c19b9f
        return map(self.sanitize_bool, params)
        #return tuple(self.sanitize_bool(param) for param in params)

    def modify_params(self, params):
        #CSOANE: Had to reverse the change from map to tuple in
        # commit 97ead19db2218ea0b3c2f19aca8f519d97c19b9f
        return map(self.sanitize_geometry, params)
        #return tuple(self.sanitize_geometry(param) for param in params)

#### What is the current behavior?
See "expected behavior" section above.

#### What are the steps to reproduce?
Follow the [Django tutorial](https://docs.djangoproject.com/en/1.10/intro/tutorial01/)  up to the `python manage.py migrate` command.

#### Which versions of Python and Django are affected?
I am using pyhdb 0.3.3, Django (1.10.5), the head of django-hana(1.1) and python 2.7.

#### Is there anything else we should know?
Here are the changes I made to settings.py:
```
#CSOANE: Changed the default database from sqllite to HANA
DATABASES = {
        'default': {
            'ENGINE': 'django_hana',
            'NAME': 'djhxepure',  # This is db schema name. It will be added if it does not exist.
            'USER': 'System',
            'PASSWORD': 'mypassword', # Replace with your password.
            'HOST': '10.125.20.57',  # Replace with your HXE server IP address
            'PORT': '39013', #I am using the SytemDB on a multi-tenant db
        }
    }

#CSOANE: Changed UZE_TZ from True to False
USE_TZ = False
```

**Comment by mathebox** [comment]
Fixed by #23

Note: `map()` in python3 returns objects of type `mapobjects` which conflict with pyhdb

**Comment by carlsoane** [comment]
That was quick! Thanks Max.
